lib/checkPullRequestLabels.js

```diff
@@ -363,3 +365,31 @@ module.exports.checkForIssueLabel = async function (context) {
     await context.github.issues.removeLabel(removeLabelParams);
   }
 };
+
+/**
+ * This function checks and prevent the removal of stale build label
+ * by unauthorised users.
+ *
+ * @param {import('probot').Context} context
+ */
+module.exports.checkBuildLabelRemoval = async function (context) {
+  /**
+   * @type {import('probot').Octokit.IssuesGetLabelResponse} label
+   */
+  const label = context.payload.label;
+  if (label.name === OLD_BUILD_LABEL) {
+    const user = context.payload.sender;
+    const commentParams = context.issue({
+      body:
+        'Hi @' + user.login + ', the build of this pr is ' +
+        ' stale please do not remove \'' + OLD_BUILD_LABEL + '\'' +
+        ' label. ',
+    });
+    await context.github.issues.createComment(commentParams);
```

Comment:
Before commenting on this PR, we need to actually check if the build of the PR is stale.

See https://github.com/oppia/oppiabot/blob/959b59c9ccc6cd1494b249f6b1fa11176258bbfc/lib/staleBuildChecks.js#L22-L30

Reply:
yes sir i agree with you i had added the check in the latest commit.